src/Aspire.Hosting.Azure/Extensions/AzureSearchExtensions.cs

```diff
@@ -71,12 +71,9 @@ void ConfigureSearch(ResourceModuleConstruct construct)
             // just public Azure in the future.  https://github.com/Azure/azure-sdk-for-net/issues/42640
             search.AddOutput("connectionString", "'Endpoint=https://${{{0}}}.search.windows.net'", me => me.Name);
 
-            if (configureResource is not null)
-            {
-                var resource = (AzureSearchResource)construct.Resource;
-                var resourceBuilder = builder.CreateResourceBuilder(resource);
-                configureResource(resourceBuilder, construct, search);
-            }
+            var resource = (AzureSearchResource)construct.Resource;
+            var resourceBuilder = builder.CreateResourceBuilder(resource);
+            configureResource?.Invoke(resourceBuilder, construct, search);
```

Comment:
(super nit) It's kind of weird to create something (the `resourceBuilder`) and then not use it when `configureResource` is null. Obviously, it doesn't really matter. But it just looks odd reading the code.